pkg/tenant/db/members.go

```diff
@@ -82,11 +92,15 @@ func (m *Member) Validate() error {
 // Convert the model to an API response
 func (m *Member) ToAPI() *api.Member {
 	return &api.Member{
-		ID:       m.ID.String(),
-		Name:     m.Name,
-		Role:     m.Role,
-		Created:  TimeToString(m.Created),
-		Modified: TimeToString(m.Modified),
+		ID:           m.ID.String(),
+		Email:        m.Email,
+		Name:         m.Name,
+		Role:         m.Role,
+		Status:       m.Status,
+		Created:      TimeToString(m.Created),
+		Modified:     TimeToString(m.Modified),
```

Comment:
One thing we might consider is removing the `Created` and `Modified` from the API struct, maybe instead of `Created` we can have `InvitedAt` (since that's a bit more useful for the frontend). Basically what I'm trying to say is that `Created` and `Modified` are more database-oriented and if we're not displaying them in the teammates table then we don't have to return them.

Reply:
Sure. I don't think `Modified` is used anywhere, but `Created` is currently used on the `/profile` page for `Date Created`.